<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIPS IDE Beta V1.1 by Lucas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Lucas/Desktop/DBP/books.ico</normaloff>C:/Users/Lucas/Desktop/DBP/books.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>输入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="maximumSize">
           <size>
            <width>567</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>输出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_3"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Build Log</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="textEdit_2"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File (&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu_O">
     <property name="title">
      <string>打开 (&amp;O)</string>
     </property>
     <addaction name="action_open_ASM"/>
     <addaction name="action_open_Bin"/>
     <addaction name="action_open_Coe"/>
    </widget>
    <widget class="QMenu" name="menu_A">
     <property name="title">
      <string>保存为 (&amp;A)</string>
     </property>
     <addaction name="action_save_ASM"/>
     <addaction name="action_save_BIN"/>
     <addaction name="action_save_COE"/>
    </widget>
    <addaction name="action_N"/>
    <addaction name="menu_O"/>
    <addaction name="action_S"/>
    <addaction name="menu_A"/>
    <addaction name="action_P"/>
    <addaction name="actionExit_X"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="action_U"/>
    <addaction name="action_T"/>
    <addaction name="action_C"/>
    <addaction name="action_P_2"/>
    <addaction name="action_L"/>
    <addaction name="action_F"/>
    <addaction name="action_R"/>
    <addaction name="action_G"/>
    <addaction name="action_A_2"/>
   </widget>
   <widget class="QMenu" name="menuBuild_B">
    <property name="title">
     <string>Build(&amp;B)</string>
    </property>
    <addaction name="action_Asm"/>
    <addaction name="actionCoe"/>
    <addaction name="action_BIN"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug(&amp;D)</string>
    </property>
    <addaction name="action_Step"/>
    <addaction name="action_Jal"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild_B"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_N">
   <property name="text">
    <string>新建 (&amp;N)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存 (&amp;S)</string>
   </property>
  </action>
  <action name="action_P">
   <property name="text">
    <string>打印 (&amp;P)</string>
   </property>
  </action>
  <action name="actionExit_X">
   <property name="text">
    <string>Exit(&amp;X)</string>
   </property>
  </action>
  <action name="action_U">
   <property name="text">
    <string>撒消 (&amp;U)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>剪切 (&amp;T)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制 (&amp;C)</string>
   </property>
  </action>
  <action name="action_P_2">
   <property name="text">
    <string>粘贴 (&amp;P)</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>删除 (&amp;L)</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>查找 (&amp;F)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>替换 (&amp;R)</string>
   </property>
  </action>
  <action name="action_G">
   <property name="text">
    <string>转到 (&amp;G)</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>全选 (&amp;A)</string>
   </property>
  </action>
  <action name="action_Asm">
   <property name="text">
    <string>汇编 (Asm)</string>
   </property>
  </action>
  <action name="actionCoe">
   <property name="text">
    <string>Coe</string>
   </property>
  </action>
  <action name="action_BIN">
   <property name="text">
    <string>反汇编</string>
   </property>
  </action>
  <action name="action_Step">
   <property name="text">
    <string>单步(Step)</string>
   </property>
  </action>
  <action name="action_Jal">
   <property name="text">
    <string>单步(跳过Jal)</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>运行到…</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="action_open_ASM">
   <property name="text">
    <string>ASM</string>
   </property>
  </action>
  <action name="action_open_Bin">
   <property name="text">
    <string>bin</string>
   </property>
  </action>
  <action name="action_open_Coe">
   <property name="text">
    <string>coe</string>
   </property>
  </action>
  <action name="action_save_ASM">
   <property name="text">
    <string>Asm</string>
   </property>
  </action>
  <action name="action_save_BIN">
   <property name="text">
    <string>Bin</string>
   </property>
  </action>
  <action name="action_save_COE">
   <property name="text">
    <string>Coe</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
